modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -0,0 +1,304 @@
+                "AWS": "arn:aws:iam::301721915996:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+
+# Trust relationships trusted entity trusted account A configuration on test-efs-acrossaccount-role in account B
+
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::301721915996:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+
+# test-cross-account-assume-policy policy attached to test-efs-acrossaccount-role in account B
+
+{
+    "Version": "2012-10-17",
+    "Statement": {
+        "Effect": "Allow",
+        "Action": "sts:AssumeRole",
+        "Resource": "arn:aws:iam::589722580343:role/test-efs-acrossaccount-role"
+    }
+}
+
+# test-efs-acrossaccount-driver-policy attached to test-efs-acrossaccount-role in account B
+
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeNetworkInterfaces",
+                "ec2:DescribeSubnets"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "elasticfilesystem:DescribeMountTargets",
+                "elasticfilesystem:DeleteAccessPoint",
+                "elasticfilesystem:ClientMount",
+                "elasticfilesystem:DescribeAccessPoints",
+                "elasticfilesystem:ClientWrite",
+                "elasticfilesystem:ClientRootAccess",
+                "elasticfilesystem:DescribeFileSystems",
+                "elasticfilesystem:CreateAccessPoint"
+            ],
+            "Resource": [
+                "arn:aws:elasticfilesystem:*:589722580343:access-point/*",
+                "arn:aws:elasticfilesystem:*:589722580343:file-system/*"
+            ]
+        }
+    ]
+}
+----
+
+. In AWS account A, attach an inline policy to the IAM role of the AWS EFS CSI driver's controller service account with the necessary permissions to perform Security Token Service (STS) assume role on the IAM role created earlier.
++
+[source, json]
+----
+# test-cross-account-assume-policy policy attached to Openshift cluster efs csi driver user in account A
+
+{
+    "Version": "2012-10-17",
+    "Statement": {
+        "Effect": "Allow",
+        "Action": "sts:AssumeRole",
+        "Resource": "arn:aws:iam::589722580343:role/test-efs-acrossaccount-role"
+    }
+}
+----
+
+. In AWS account A, attach the AWS-managed policy “AmazonElasticFileSystemClientFullAccess” to {product-title} cluster master role. The role name is in the form `<clusterID>-master-role` (for example, `test-0120ef-czjrl-master-role`).
+
+. Create a Kubernetes secret with "awsRoleArn" as the key and the role created earlier as the value:
```

Comment:
Suggestion: replace doublequotes with backtick: `awsRoleArn` to indicate a value to be included in the oc command.